tests/cupy_tests/linalg_tests/test_decomposition.py

```diff
@@ -89,12 +91,19 @@ def check_mode(self, array, mode, dtype):
     @testing.fix_random()
     @condition.repeat(3, 10)
     def test_mode(self):
-        self.check_mode(numpy.random.randn(0, 3), mode=self.mode)
-        self.check_mode(numpy.random.randn(3, 0), mode=self.mode)
         self.check_mode(numpy.random.randn(2, 4), mode=self.mode)
         self.check_mode(numpy.random.randn(3, 3), mode=self.mode)
         self.check_mode(numpy.random.randn(5, 4), mode=self.mode)
 
+    @testing.fix_random()
+    @condition.repeat(3, 10)
+    @unittest.skipUnless(
+        LooseVersion(numpy.__version__) > LooseVersion('1.16.0'),
+        'NumPy<1.16 does not work with empty arrays')
```

Comment:
There is a decorator for this: `cupy.testing.with_requires`.
Let's use it.

Reply:
Done.